pkg/webui/components/form/index.js

```diff
@@ -134,9 +141,18 @@ class InnerForm extends React.Component {
             })
           }
         } else if (Child.type === FieldGroup) {
+          const { name } = Child.props
+          const value = name && getByPath(values, name)
```

Comment:
`name` is the required prop for the `<FieldGroup />` component. no need for conditional assignment here.